subsys/bluetooth/audio/capabilities.c

```diff
@@ -298,6 +301,28 @@ static int write_location_cb(struct bt_conn *conn, enum bt_audio_dir dir,
 #endif /* CONFIG_BT_PAC_SNK_LOC_WRITEABLE || CONFIG_BT_PAC_SRC_LOC_WRITEABLE */
 #endif /* CONFIG_BT_PAC_SNK_LOC || CONFIG_BT_PAC_SRC_LOC */
 
+static int get_available_contexts_cb(struct bt_conn *conn, enum bt_audio_dir dir,
+				     enum bt_audio_context *contexts)
+{
+	IF_ENABLED(CONFIG_BT_PAC_SNK, (
+		if (dir == BT_AUDIO_DIR_SINK) {
+			*contexts = sink_available_contexts;
+			return 0;
+		}
+	));
+
+	IF_ENABLED(CONFIG_BT_PAC_SRC, (
+		if (dir == BT_AUDIO_DIR_SOURCE) {
+			*contexts = source_available_contexts;
+			return 0;
+		}
+	));
+
+	BT_ERR("Invalid endpoint dir: %u", dir);
+
+	return -EINVAL;
```

Comment:
Should `contexts` be set to 0 in that case? 

Should we really return an error here, as both the available and supported contexts always contains both sink and source contexts?

Reply:
I guess not, as the invalid direction is stack fault. Thus I'll put an assert here instead of debug log.